### DISCUSSION TITLE
Links on entries overview acting not the same as on an entry page

### DISCUSSION THREAD

**Discussion by bavocorteville** [craftcms/cms#10506] [Ideas]
### Description
The 'Primary entry page' view link takes into account whether the entry/site is disabled (by adding x-craft-preview and/or siteToken param). The link in the overview of the entry (globe icon) does not. 

### Additional info

- Craft version: 3.7.29
- PHP version: 7.4.27
- Database driver & version: MySQL 8.0.19

**Comment by timkelty** [comment]
🤔 This is expected behavior as things stand now, as the two cases are inherently different…the globe is a static representation of the elements URL, while the view/preview buttons are actively creating a tokenized view of an entry.

That said, I can see how having a tokenized preview link accessible from the index would be desirable as well.

Converting to a discussion for futher…discussion.
